app/views/shared/_navigation.html.erb

```diff
@@ -53,7 +53,9 @@
         <% if current_user.teams.length > 0 %>
           <li id="team-switch">
             <div class="btn-group">
-              <button type="button" class="btn btn-default"><%= truncate_team_name(current_team.name) %></button>
+              <button type="button" class="btn btn-default" data-toggle="dropdown">
```

Comment:
![image](https://user-images.githubusercontent.com/15074090/60701136-ec996800-9efa-11e9-9235-a0014a1a184b.png)
If you repeatedly click on it, the text start to disappear.
![image](https://user-images.githubusercontent.com/15074090/60701159-06d34600-9efb-11e9-92dc-c4dd2b06953d.png)

Reply:
I think this bug not correspond to this issue, and must be fixed in different task. I will create issue.